### PR TITLE
Add feature which executes additional commands when switching

### PR DESCRIPTION
**Pull request by niecore**
New section "commands" is added to the configuration
List of commands for "guest" and "host" can be set.

Example:

commands:
  guest:
    - echo switch to guest
  host:
    - echo switch to host

edit:

This can be useful when you want to change the screen setup via `xrandr` or similiar.

### CODE REVIEW

**Review by alexbakker — CHANGES_REQUESTED** [review]
Good idea! Two suggestions:

1. Can you change the subprocess call to ``subprocess.call(command, shell=True)``? This'll offer users a bit more flexibility.
2. Could you also make the "commands" block optional, so that users have the option to disable that functionality?

**Comment by niecore** [comment]
I addressed the requested changes.

File: virtkvm/__init__.py [hunk 1]
```diff
@@ -107,14 +113,22 @@ def _call_dccutil(display: dict, ident: int):
             "setvcp", hex(display["feature"]), hex(ident)
         ])
 
+    @staticmethod
+    def _call_commands(command: str):
+        return subprocess.call(command.split(" "), shell=True)
```
Comment (alexbakker) [inline virtkvm/__init__.py]: Now that ``shell=True``, the command should not be split.

**Review by alexbakker — CHANGES_REQUESTED** [review]
One more minor change.

**Comment by alexbakker** [comment]
Thanks a lot!

File: virtkvm/__init__.py (hunk 1 above)
Comment (niecore) [reply virtkvm/__init__.py]: Okay, now I understand the difference with the parameter `shell`.

Thanks for the review and the advises :)

**Pull request merged**